tests/ZendService/Amazon/SimpleDb/OnlineTest.php

```diff
@@ -364,23 +364,23 @@ public function testListDomains()
             // Amazon returns an empty page as the last page :/
             $isLast = $page->isLast();
             if (!$isLast) {
-              // The old isLast() assertTrue failed in full suite runs. Token often
+                // The old isLast() assertTrue failed in full suite runs. Token often
               // decodes to 'TestsZendServiceAmazonSimpleDbDomain_testPutAttributes'
```

Comment:
Looks like it properly indented the line below the conditional, but the next 5 lines were not indented correctly.